huaweicloud/data_source_huaweicloud_kms_key_v1.go

```diff
@@ -173,7 +173,6 @@ func dataSourceKmsKeyV1Read(d *schema.ResourceData, meta interface{}) error {
 	d.Set("scheduled_deletion_date", key.ScheduledDeletionDate)
 	d.Set("key_state", key.KeyState)
 	d.Set("default_key_flag", key.DefaultKeyFlag)
-	d.Set("expiration_time", key.ExpirationTime)
```

Comment:
It's better to add the `expiration_time` into schema rather than deleting it

Reply:
done